usecases/objects/references_delete.go

```diff
@@ -59,12 +58,34 @@ func (m *Manager) DeleteObjectReference(ctx context.Context, principal *models.P
 		}
 	}
 
+	res, err := m.getObjectFromRepo(ctx, input.Class, input.ID, additional.Properties{}, nil, tenant)
+	if err != nil {
+		errnf := ErrNotFound{}
+		if errors.As(err, &errnf) {
+			return &Error{"source object", StatusNotFound, err}
+		} else if errors.As(err, &ErrMultiTenancy{}) {
+			return &Error{"source object", StatusUnprocessableEntity, err}
+		}
```

Comment:
```suggestion
		if errors.As(err, &ErrNotFound{}) {
			return &Error{"source object", StatusNotFound, err}
		} else if errors.As(err, &ErrMultiTenancy{}) {
			return &Error{"source object", StatusUnprocessableEntity, err}
		}
```

We could remove `errf` variable if it's not used anywhere else